data/mods/Magiclysm/items/alchemy_items.json

```
@@ -301,5 +301,125 @@
     "bashing": 10,
     "to_hit": 1,
     "category": "spare_parts"
+  },
+  {
+    "id": "alchemist_bomb",
+    "type": "TOOL",
+    "category": "weapons",
+    "name": { "str": "Мana bomb" },
+    "description": "Mana bombs are powerful explosive devices developed by a little-known alchemist group for use against enemy spellcasters.  These bombs are fueled by pure arcane energy and can kill instantly.  The exact workings are unclear, but they appear to use some kind of massive mana burn.",
+    "weight": "430 g",
+    "volume": "200 ml",
+    "price": 0,
+    "price_postapoc": 0,
+    "to_hit": -1,
+    "bashing": 14,
+    "material": [ "steel", "plastic" ],
```

```suggestion
    "material": [ "steel", "concentrated_mana" ],
```
In-line with the others.

---

```suggestion
    "material": [ "crystallized_mana", "concentrated_mana" ],
```